.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    ignore:
+      - dependency-name: "actions/*"
+      - update-types: "version-update:semver-minor"
```

Comment:
I have not tried this line before, usually I ignore `actions/*`, since GitHub is good at providing moving "simple" tags and it's pretty rare (though it has happened) that an update to these breaks things.

Reply:
Ignoring minor updates... so does this mean dependabot will only update to newer patch versions? And newer major versions?

Reply:
Actually, I didn't structure it correctly anyway: https://github.blog/changelog/2021-05-21-dependabot-version-updates-can-now-ignore-major-minor-patch-releases/

Reply:
Ah, I see, that's nested within the `actions/*` rule... and Github actions will move the `v2` tag to point to the latest `v2` e.g. `v2.3.4`... how strange! Well, anyway, that's cool with me.